test/testinput/bufr_ncep_highRes_sonde.yaml

```diff
@@ -0,0 +1,209 @@
+          units: "m"
+
+        #- name: "MetaData/stationIdentification"
+          #coordinates: "longitude latitude"
+          #source: variables/stationIdentification
+          #longName: "Station Identification"
+
+        #- name: "ObsValue/cloudType"
+          #coordinates: "longitude latitude"
+          #source: variables/cloudType
+          #longName: "Cloud Type"
+
+        # ObsValue
+        - name: "ObsValue/pressure"
+          coordinates: "longitude latitude"
+          source: variables/pressure
+          longName: "Pressure"
+          units: "Pa"
+
+        - name: "ObsValue/airTemperature"
+          coordinates: "longitude latitude"
+          source: variables/airTemperature
+          longName: "Air Temperature"
+          units: "K"
+
+        - name: "ObsValue/dewpointTemperature"
+          coordinates: "longitude latitude"
+          source: variables/dewpointTemperature
+          longName: "Dewpoint Temperature"
+          units: "K"
+
+        - name: "ObsValue/windDirection"
+          coordinates: "longitude latitude"
+          source: variables/windDirection
+          longName: "Wind Direction"
+          units: "degree"
+
+        - name: "ObsValue/windSpeed"
+          coordinates: "longitude latitude"
+          source: variables/windSpeed
+          longName: "Wind Speed"
+          units: "m s-1"
+
+        # ObsValue - ocean
+        - name: "ObsValue/waterTemperature"
+          coordinates: "longitude latitude"
+          source: variables/waterTemperature
+          longName: "Water Temperature"
+          units: "K"
+
+        # ObsValue - cloud, visibility, gust wind, min/max temperature, weather
+        #- name: "ObsValue/cloudAmount"
+          #coordinates: "longitude latitude"
+          #source: variables/cloudAmount
+          #longName: "Description of Cloud Amount"
+
+        #- name: "ObsValue/heightOfBaseOfCloud"
+          #coordinates: "longitude latitude"
+          #source: variables/heightOfBaseOfCloud
+          #longName: "Cloud Base Altitude"
+          #units: "m"
+
+        #- name: "ObsValue/verticalSignificanceSurfaceObservations"
+          #coordinates: "longitude latitude"
+          #source: variables/verticalSignificanceSurfaceObservations
+          #longName: "Description of Vertical Significance (Surface Observations)"
+
+        # QualityMarker
+        - name: "QualityMarker/stationElevation"
+          coordinates: "longitude latitude"
+          source: variables/stationElevationQM
+          longName: "Quality Indicator for Station Elevation"
+
+        - name: "QualityMarker/pressure"
+          coordinates: "longitude latitude"
+          source: variables/pressureQM
+          longName: "Quality Indicator for Pressure"
+
+        - name: "QualityMarker/airTemperature"
+          coordinates: "longitude latitude"
+          source: variables/airTemperatureQM
+          longName: "Quality Indicator for Temperature"
+
+        - name: "QualityMarker/moisture"
```

Comment:
Recommend changing "moisture" to whatever variable is actually used in the code. Just to be more precise.

Reply:
Changing it would also decrease the number of warnings

Reply:
agree @nicholasesposito shouldn't this be:
`QualityMarker/dewPointTemperature` for consistency with the `ObsValue`

Reply:
@nicholasesposito @BenjaminRuston agreed, done. Thanks.